APCAppCore/APCAppCore/DataSubstrate/Model/APCUser+Bridge.m

```diff
@@ -642,10 +642,11 @@ - (void) showTestUserVerificationAlertWithVc:(__weak UIViewController*)vc
     
     void (^showVcBlock)() = ^
     {
-        NSString* yesStr = NSLocalizedString(@"YES", @"Positive Answer");
-        NSString* noStr =  NSLocalizedString(@"NO", @"Negative Answer");
-        NSString* title =  NSLocalizedString(@"Are you a tester?", @"Question if the user is a quality assurance tester");
-        NSString* msg = [NSString stringWithFormat:NSLocalizedString(@"Based on your email address, we have detected you are a tester for %@.  If this is correct, select %@ so we can store your data separately.", @"Message informing user if and what happens if they are a tester"), [APCUtilities appName], [yesStr lowercaseString]];
+        NSString* yesStr = NSLocalizedStringWithDefaultValue(@"YES", @"APCAppCore", APCBundle(), @"YES", @"Positive Answer");
```

Comment:
As long as you're fixing this, can we use localization keys that aren't just the English text, and not make the button titles shout in ALL CAPS?